pkg/console/starter/starter.go

```diff
@@ -118,6 +128,11 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		oauthinformers.WithTweakListOptions(tweakListOptionsForOAuth),
 	)
 
+	consoleInformer := consoleinformers.NewSharedInformerFactory(
```

Comment:
Prob plural like the others `consoleInformers`.

Reply:
Then need to ensure we start the informers, this one is new:

```
	for _, informer := range []interface {
		Start(stopCh <-chan struct{})
	}{
                // etc.
		resourceSyncerInformers,
		operatorConfigInformers,
                // add here, and should be good!
		consoleInformers,
		configInformers,
                 // etc.
	} {
		informer.Start(ctx.Done())
	}
```

Reply:
Yup, start the informer and you will get this:

![Screen Shot 2019-10-07 at 12 37 38 PM](https://user-images.githubusercontent.com/280512/66330914-5e6cbd80-e8ff-11e9-9ca9-afe4ecacc0da.png)